core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -105,6 +105,10 @@
                                   style="margin-left: 10px;">
         </exploration-summary-tile>
       </div>
+
+      <div ng-if="collectionSummary !== null" class="conversation-skin-back-to-collection-container">
```

Comment:
I think this does the job, because collectionId only exists when collectionSummary isn't null, from what I understand.
@seanlip Please let me know if this is wrong.